gamemode/gamemode-1.7.json

```
@@ -13,5 +13,11 @@
       "sha256": "8a1718c657cb1d3c8269edfd397faf3cb41fccd885198ef0225bf8f82217c42f"
     }
   ],
-  "cleanup": [ "/include", "/lib/pkgconfig", "/lib/libgamemodeauto.a" ]
+  "cleanup": [ "/include", "/lib/pkgconfig" ],
+  "post-install": [
+    "mkdir -p /app/bin/",
+    "mv ../data/gamemoderun /app/bin/gamemoderun",
+    "chmod +x /app/bin/gamemoderun",
```

```suggestion
    "install -t /app/bin -Dm755 ../data/gamemoderun",
```

This probably works.